plugin.video.kla.tv/addon.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version='1.0' encoding='UTF-8' standalone='yes'?>
+<addon 
+  id="plugin.video.kla.tv" 
+  name="Klagemauer TV" 
+  provider-name="celadoor" 
+  version="0.0.4">
+  <requires>
+    <import  addon="xbmc.python"                      version="2.1.0" />
+    <import  addon="script.common.plugin.cache"       version="1.5.1" />
+    <import  addon="script.module.parsedom"           version="1.5.1" />
+    <import  addon="script.module.simple.downloader"  version="0.9.4" />
+  </requires>
+  <extension library="addon.py"                     point="xbmc.python.pluginsource">
+    <provides>video</provides>
+  </extension>
+  <extension point="xbmc.addon.metadata">
+    <summary lang="de">www.kla.tv video plugin</summary>
```

Comment:
please also adjust the language tags to use "language-country" naming scheme. See here for more information: http://kodi.wiki/view/language_support